manifests/qpid.pp

```diff
@@ -1,8 +1,9 @@
-# Katello Config
+# Katello qpid Config
 class katello::qpid (
-  $client_cert            = undef,
-  $client_key             = undef,
-  $katello_user           = $katello::user
+  $client_cert,
+  $client_key,
+  $katello_user          = $::katello::user,
+  $candlepin_event_queue = $::katello::candlepin_event_queue,
```

Comment:
Right now the katello class inherits from params so that way it should be available. By using it this way you can easily make it a real parameter later on.